docs/source/conf.py

```diff
@@ -40,6 +40,7 @@
               'sphinx.ext.intersphinx',
               'sphinxemoji.sphinxemoji',
               'sphinx_reredirects',
+              'qiskit_sphinx_theme',
```

Comment:
This activates jQuery, which is no longer done by default with Sphinx 6. The Pytorch theme requires jQuery.